actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -173,11 +173,14 @@ def _back_url # :nodoc:
       #   link_to "Visit Other Site", "http://www.rubyonrails.org/", data: { confirm: "Are you sure?" }
       #   # => <a href="http://www.rubyonrails.org/" data-confirm="Are you sure?">Visit Other Site</a>
       #
-      # Also you can set target attribute with <tt>target</tt> options:
+      # Also you can set any link attributes such as <tt>target</tt>, <tt>rel</tt>, <tt>type</tt>
+      # using these HTML options:
       #
-      #   link_to "External link", "http://www.rubyonrails.org/", target: "_blank"
-      #   # => <a href="http://www.rubyonrails.org/" target="_blank">External link</a>
+      #   link_to "External link", "http://www.rubyonrails.org/", target: "_blank", rel: "nofollow"
+      #   # => <a href="http://www.rubyonrails.org/" target="_blank" rel="nofollow">External link</a>
       #
+      #   link_to "Read the articles", controller: "articles", type: "text/html"
+      #   # => <a href="/articles" type="text/html">Read the articles</a>
```

Comment:
I think we don't need two examples, just stating that we can pass the extra options and one example like the `rel` one should be enough. Wdyt?